guides/credential-ux-guide.md

```diff
@@ -54,13 +54,13 @@ The Meta Overlay data in the OCA file describes the schema to the wallet, which
-| credential_support_url | A link to a web page that has help information for users of the credential | *Not used* | <ul><li>MUST have a value<ul><li>*Include in anticipation of when BC Wallet users this attribute*</li></ul></li><li>URL for a service providing support in the use of the credential</li><li>Should link to a support page maintained by the program area (possibly a division or branch within the issuing organization) responsible for the credential</li></ul> |
+| name | The name of the credential | Shown on card view | - MUST have a value<br> - The name of the credential<br> - Should be short – 1-3 words<br> - Should be the name of the Schema on the ledger<br> - Examples: Person, Digital Business Card |
+| description | A plain language definition of the credential | *Not used* | - MUST have a value - *Include in anticipation of when BC Wallet uses this attribute*<br>A description of the credential<br> - Should be short – 10-20 words<br> - Should reflect the subject/topic of the credential, i.e., what the credential attributes describe |
+| issuer | The name of the organization that issues the credential | Shown on card view | - MUST have a value<br> - The name of the issuer of the credential<br> - Should be the legal name of the organization, not a department or branch within the organization - For credentials issued directly by the BC Government, this MUST be “Government of British Columbia”, unless a legislated role (or similar) is more appropriate<br> - For credentials issued by crown corporations, this MUST be the name of corporation, e.g., for BC Hydro, this should be “British Columbia Hydro and Power Authority“ |
+| issuer_description | A description for the issuer of the credential | *Not used* | - *This should be left blank until how it is used by the BC Wallet is defined* |
+| issuer_url | A URL for the issuer of the credentialA URL for the issuer of the credential | *Not used* | - *This should be left blank until how it is used by the BC Wallet is defined* |
+| credential_help_text | Help text that is presented to the user of the BC Wallet in the context of the credential <br>If credential_support_url is defined, this text will be a hyperlink or button that a person can tap, resulting in the web page referenced by  credential_support_url being opened on the device browser | *Not used* | - *Include in anticipation of when BC Wallet users this attribute *<br> - Should be concise and specific to the credential<br> - If it will present as a hyperlink, this should be short – 3-5 words and indicate where the URL goes (e.g., “Digital Business Card help page”) |
```

Comment:
Remove space at the end of: *Include in anticipation of when BC Wallet users this attribute *